client/src/components/container/admin/LogInPage/style.css

```diff
@@ -0,0 +1,129 @@
+.container{
+    display: flex;
+    overflow: hidden;
+}
+
+
+.main-section{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    align-self: center;
+    justify-content: center;
+    height: 100%;
+    width: 50%;
+}
+
+.logo{
+    width:230px;
+}
+
+.main-header{
+    color:#FB6C04;
+    font-size: 45px;
+}
+
+.sub-header{
+    color: #186AA5;
+    margin-bottom: 20px;
+    font-size: 25px;
+}
+
+.span{
+    color: #186AA5;
+}
+
+.img-container{
+    width: 50%;
+    overflow: hidden;
+}
+
+.main-img{
+    width: 100%;
+    height: 100vh;
+    display: block;
+}
+
+
+.main-section .ant-input-affix-wrapper{
+    height: 47px !important;
```

Comment:
There's two ways to override these rules, either with `!important` or a css rule with higher specificity, its better to use the second if possible